authentication/src/main/java/io/camunda/authentication/session/WebSessionRepository.java

```diff
@@ -58,15 +58,19 @@ public void save(final WebSession webSession) {
   @Override
   public WebSession findById(final String id) {
     LOGGER.debug("Retrieve session {}", id);
-    return Optional.ofNullable(persistentWebSessionClient.getPersistentWebSession(id))
+    return Optional.ofNullable(id)
+        .map(i -> isSessionIdEmpty(i) ? i : null)
+        .map(persistentWebSessionClient::getPersistentWebSession)
         .map(this::getWebSessionIfNotExpired)
         .orElse(null);
```

Comment:
I think we can make it simpler this way
```suggestion
    if (isSessionIdEmpty(id)) {
        return null;
    }
    return Optional.ofNullable(persistentWebSessionClient.getPersistentWebSession(id))
        .map(this::getWebSessionIfNotExpired)
        .orElse(null);
```